### ISSUE TITLE
Why do we need vocabulary_counts in Skipgram?

### ISSUE THREAD

**Issue by phucdoitoan** [phanein/deepwalk#111]
Hi, I try to figure out the role of vocabulary_counts in skipgram.py, but it seems that the variable self.vocabulary_counts does not have any effect on the codes. 
In the Word2Vec implementation https://github.com/RaRe-Technologies/gensim/blob/develop/gensim/models/word2vec.py there is no use for vocabulary_counts neither. 

Is it just a variable of an older version that is not necessary now?

**Comment by GTmac** [comment]
Yeah you are right! The skipgram.py implementation is based on a rather old version of Gensim word2vec; it is not necessary anymore. Would you like to clean this up? Happy to help with reviewing this. Thank you!
